tests/test_views.py

```diff
@@ -22,16 +25,24 @@ def post(self, request, *args, **kwargs):
         return Response({'method': 'POST', 'data': request.data})
 
 
+class BasicAsyncView(APIView):
+    async def get(self, request, *args, **kwargs):
+        return Response({'method': 'GET'})
+
+    async def post(self, request, *args, **kwargs):
+        return Response({'method': 'POST', 'data': request.data})
+
+
 @api_view(['GET', 'POST', 'PUT', 'PATCH'])
 def basic_view(request):
     if request.method == 'GET':
-        return {'method': 'GET'}
+        return Response({'method': 'GET'})
     elif request.method == 'POST':
-        return {'method': 'POST', 'data': request.data}
+        return Response({'method': 'POST', 'data': request.data})
     elif request.method == 'PUT':
-        return {'method': 'PUT', 'data': request.data}
+        return Response({'method': 'PUT', 'data': request.data})
     elif request.method == 'PATCH':
-        return {'method': 'PATCH', 'data': request.data}
+        return Response({'method': 'PATCH', 'data': request.data})
```

Comment:
Do these need to be changed?

Reply:
Yes, because otherwise tests fail since `APIView` is expecting a `Response` object. Before there were no tests for get and post and that's why the issue was never encountered before.